hpobench/benchmarks/mo/lm_benchmark.py

```diff
@@ -0,0 +1,368 @@
+        print("len of corpus dict", self.ntokens)
+
+    def __seed_everything(self):
+        """Helperfunction: Make the benchmark deterministic by setting the correct seeds"""
+        seed = self.rng.randint(0, 100000)
+        print("seed obtained", seed)
+        random.seed(seed)
+        np.random.seed(seed)
+        torch.manual_seed(seed)
+        torch.cuda.manual_seed_all(seed)
+        torch.backends.cudnn.deterministic = True
+
+    @staticmethod
+    def get_configuration_space(seed: Union[int, None] = None) -> CS.ConfigurationSpace:
+        """Parameter space to be optimized --- contains the hyperparameters
+        """
+        cs = CS.ConfigurationSpace(seed=seed)
+
+        cs.add_hyperparameters([
+            CS.UniformIntegerHyperparameter(
+                'batch_size', default_value=128, lower=8, upper=256
+            ),
+            CS.UniformIntegerHyperparameter(
+                'emsize', default_value=128, lower=32, upper=1024, log=True
+            ),
+            CS.UniformIntegerHyperparameter(
+                'lr_factor', default_value=50, lower=1, upper=100, log=True
+            ),
+            CS.UniformFloatHyperparameter(
+                'lr', default_value=5, lower=1, upper=50, log=True
+            ),
+            CS.UniformFloatHyperparameter(
+                'dropout', default_value=0.99, lower=0, upper=0.99
+            ),
+            CS.UniformFloatHyperparameter(
+                'clip', default_value=0.99, lower=0.1, upper=2
+            )
+
+        ])
+        return cs
+
+    @staticmethod
+    def get_objective_names(self) -> List[str]:
+        return ['log_perplexity', 'accuracy', 'time']
+
+    @staticmethod
+    def get_fidelity_space(seed: Union[int, None] = None) -> CS.ConfigurationSpace:
+
+        fidelity_space = CS.ConfigurationSpace(seed=seed)
+        fidelity_space.add_hyperparameters([
+            CS.UniformIntegerHyperparameter(
+                'budget', lower=1, upper=81, default_value=81, log=False
+            )
+        ])
+        return fidelity_space
+
+    @staticmethod
+    def get_meta_information() -> Dict:
+        """ Returns the meta information for the benchmark """
+        return {
+            'name': 'Multi-objective Asynchronous Successive Halving',
+            'references': ['@article{schmucker2021multi,'
+                           'title={Multi-objective Asynchronous Successive Halving},'
+                           'author={Schmucker, Robin and Donini, Michele and Zafar, Muhammad Bilal and Salinas,'
+                           ' David and Archambeau, C{\'e}dric},'
+                           'journal={arXiv preprint arXiv:2106.12639},'
+                           'year={2021}',
+                           ],
+        }
+
+    def init_model(self, config: Union[CS.Configuration, Dict]):
+        """ Function that returns the model initialized based on the configuration and fidelity
+        """
+
+        if isinstance(config, CS.Configuration):
```

Comment:
actually we don't need that check (`if isinstance`) here. 

The objective_function -wrapper casts configurations always to dicts.